README.rst

```diff
@@ -43,6 +43,24 @@ Credential
 Support `AWS CLI credentials
 configuration <http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html>`__.
 
+
```

Comment:
Thanks for the appropriate advice.
I will swap the Credentials section and Usage section of the README.